cypress/integration/userCanReadASpecificArticle.feature.js

```diff
@@ -1,105 +1,103 @@
-describe("User can", () => {
+describe('User can', () => {
   beforeEach(() => {
-    cy.server();
+    cy.server()
     cy.route({
-      method: "GET",
-      url: "http://localhost:3000/api/articles",
-      response: "fixture:articles_data.json",
-    });
-  });
+      method: 'GET',
+      url: 'http://localhost:3000/api/articles',
+      response: 'fixture:articles_data.json',
+    })
+  })
 
-  describe("successfully", () => {
+  describe('successfully', () => {
     beforeEach(() => {
-      cy.server();
+      cy.server()
       cy.route({
-        method: "GET",
-        url: "http://localhost:3000/api/articles/1",
-        response: "fixture:single_article.json",
-      });
+        method: 'GET',
+        url: 'http://localhost:3000/api/articles/1',
+        response: 'fixture:single_article.json',
+      })
       cy.route({
-        method: "POST",
-        url: "http://localhost:3000/api/auth",
-        response: "fixture:user_can_register.json",
-      });
+        method: 'POST',
+        url: 'http://localhost:3000/api/auth/sign_in',
+        response: 'fixture:user_can_login.json',
+      })
```

Comment:
Good thinking of changing the way the user is authenticating himself in the test for efficiency!